lib/Resource.js

```diff
@@ -38,7 +38,14 @@ var Resource = function(options) {
   this.search = options.search;
   this.sort = options.sort;
 
+  this.associationOptions = {
+    removeForeignKeys: false
+  };
+
   if (options.associations) {
```

Comment:
I think this should be ```!!options.associations``` so we consider all potentially falsey values (now that options.associations can be an object or a boolean)